### PR TITLE
Use cases added for Wallet and NFT marketplace queries

### PR DESCRIPTION
**Pull request by johnnygreeney**
- divided up the queries into 12 sub-queries for 'Wallet' and 6 sub-queries under 'NFT Marketplace'
- updated the _meta.json to reflect this
- added images where necessary
- checked link and table formats on localhost, should publish correctly (although I can't preview)

### CODE REVIEW

File: docs/pages/_meta.json
Comment (satran004) [inline docs/pages/_meta.json:10]: @darlisagc @johnnygreeney 

Should we create a top-level menu called 'Use Cases' at the same level as 'Applications' and then add a subfolder for each use case (use-case1, use-case2, etc.)?

Comment (johnnygreeney) [reply docs/pages/_meta.json:10]: Screenshot 2024-09-03 at 13 34 25

i was thinking more like this, as otherwise would need 3 levels of subfolders.

Comment (johnnygreeney) [reply docs/pages/_meta.json:10]: Screenshot 2024-09-03 at 13 57 49

..alternative was to have all 12 queries on one page, but then it was a little too long.

**Review by johnnygreeney — COMMENTED** [review]
@satran004 i've update this file as suggested

**Comment by johnnygreeney** [comment]
Screenshot 2024-09-04 at 08 17 34

I will go ahead with this layout, using 'cards' also...I will need to update this PR later today

File: docs/pages/use-cases/nftqueries/nft4.mdx
Comment (darlisagc) [inline docs/pages/use-cases/nftqueries/nft4.mdx:94]: Maybe we can rephrase it? 
Let's use  address `addr1qyvl8lmwpfq7pahuadpjwul9xlj7c0atr5hvu4r3cjsjlfua276aqezvwjsfucvq76welxmytfyt7ug5835wntzzdvgsyxh7q9` as an example for this query

File: docs/pages/use-cases/nftqueries/nft4.mdx
Comment (darlisagc) [inline docs/pages/use-cases/nftqueries/nft4.mdx:185]: Not only Block, right?

File: docs/pages/use-cases/nftqueries/nft5.mdx
Comment (darlisagc) [inline docs/pages/use-cases/nftqueries/nft5.mdx:118]: Most?

File: docs/pages/use-cases/nftqueries/nft6.mdx
Comment (darlisagc) [inline docs/pages/use-cases/nftqueries/nft6.mdx:16]: Amount of ADA charged or offered?
ADA - Capital case.

File: docs/pages/use-cases/nftqueries/nft6.mdx
Comment (darlisagc) [inline docs/pages/use-cases/nftqueries/nft6.mdx:29]: Maybe here we can point the user to the example which we have the mentioned queries?

**Review by darlisagc — COMMENTED** [review]
Thank you for the hard work John. As f now, I have just review the text itself, nothing related to design.  I have suggested few edits, but there are small things to update.

**Review by darlisagc — CHANGES_REQUESTED** [review]
Thank you for all the changes done John. 
I have added just more few adjustments before approval

**Review by darlisagc — APPROVED** [review]

**Pull request merged**
